:poop: A great piece to start with

AnkeAnke opened the pull request:
There needs to be a start, and any start is better than none.
Also, top sneaky.

AnkeAnke committed:
:poop: A great piece to start with

Wumpf commented on README.md:
```diff
@@ -1,3 +1,9 @@
+
+## Pawns
+
+### :poop: Not  pawn
```
```suggestion
### :poop: Not a pawn
```

AnkeAnke replied:
A very good point, might have made that myself if I was sitting next to you and pointed the exact thing out.

gaetjen replied:
Level 4 headers for single pieces pls

gaetjen replied:
nevermind, this isn't the main.md

AnkeAnke committed:
:poop: Now not a pawn, instead of not pawn

There was an 'a' added - an 'a' and  space, to be precise. Adds clarity without the need for many characters.

Co-authored-by: Wumpf <r_andreas2@web.de>

Wumpf reviewed (commented):
@gaetjen @Gnaarf I'm ok with this pr, but I think it violates the current (weird) convention that most info is in `main.md`. Can we schedule a meeting to go over this a bit more? Thx.
